genesyscloud/resource_genesyscloud_flow_test.go

```diff
@@ -349,6 +400,9 @@ func TestAccResourceFlowSubstitutions(t *testing.T) {
 		},
 		CheckDestroy: testVerifyFlowDestroyed,
 	})
+
+	//Clean up the temporary file
+	removeFile(destFile)
```

Comment:
Using `defer` here and moving this up next to copyFile at the top of the test case could be a good idea. More assurance that the removeFile func will executed this way, after everything else.

Reply:
Nice catch.  Done.